elide-standalone/src/main/java/com/yahoo/elide/standalone/Util.java

```diff
@@ -51,30 +61,66 @@ public static EntityManagerFactory getEntityManagerFactory(String modelPackageNa
             options.put("javax.persistence.jdbc.url", "jdbc:mysql://localhost/elide?serverTimezone=UTC");
             options.put("javax.persistence.jdbc.user", "elide");
             options.put("javax.persistence.jdbc.password", "elide123");
+
+            //Bind entity classes from classpath to Persistence Unit
+            ArrayList<Class> loadedClasses = new ArrayList<>();
+            loadedClasses.addAll(ClassScanner.getAnnotatedClasses(Entity.class));
+
+            options.put(AvailableSettings.LOADED_CLASSES, loadedClasses);
         }
 
-        PersistenceUnitInfo persistenceUnitInfo = new PersistenceUnitInfoImpl("elide-stand-alone",
-                combineModelEntities(modelPackageName, includeAsyncModel), options);
+        PersistenceUnitInfo persistenceUnitInfo = null;
+
+        if (includeDynamicModel) {
+            initDynamicConfig(dynamicConfigPath);
+            Collection<ClassLoader> classLoaders = new ArrayList<>();
+            classLoaders.add(dynamicEntityCompiler.getClassLoader());
+            options.put(AvailableSettings.CLASSLOADERS, classLoaders);
 
-        return new EntityManagerFactoryBuilderImpl(
-                new PersistenceUnitInfoDescriptor(persistenceUnitInfo), new HashMap<>())
-                .build();
+            persistenceUnitInfo = new PersistenceUnitInfoImpl("elide-stand-alone",
+                    combineModelEntities(dynamicEntityCompiler, modelPackageName,
+                            includeAsyncModel, includeDynamicModel),
```

Comment:
Instead of creating the PersistentUnitInfoImpl in the if/else - this can be simplified by just setting the classloader and list of packages to include in the if/else - and then using those to create the PersistentUnitInfo outside the control block.